code/modules/atmospherics/machinery/atmospherics.dm

```diff
@@ -298,8 +298,10 @@ Pipelines + Other Objects -> Pipe network
 		else if(target_move.can_crawl_through())
 			if(returnPipenet() != target_move.returnPipenet())
 				user.update_pipe_vision(target_move)
+			var/old_loc = user.loc
 			user.loc = target_move
-			user.client.eye = target_move //if we don't do this, Byond only updates the eye every tick - required for smooth movement
+			user.client.eye = target_move // if we don't do this, Byond only updates the eye every tick - required for smooth movement
+			SEND_SIGNAL(user, COMSIG_MOVABLE_MOVED, old_loc, target_move, get_dir(old_loc, target_move))
```

Comment:
Unsure about the consequences of this, please check that any potential subscribers to this signal handle vent crawling fine. (e.g. waddling, squeaking components etc.)

Reply:
From what I can see, the only subscriber that will be hit in a notable way (beyond orbiting, of course), is jestosterone. If you ventcrawl with jestosterone, it squeaks every time you change pipes.

Given how rare jestosterone is, and given how rarely humans with jestosterone would be able to ventcrawl, I think that's an acceptable consequence